.travis.yml

```diff
@@ -5,6 +5,9 @@ install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.t
 script: |
   bash -ex .travis-opam.sh
 env:
+  global:
+  - EXTRA_DEPS="dot-merlin-reader"
+  - PINS="dot-merlin-reader.dev:. merlin.3.3.7:. dune:https://github.com/voodoos/dune.git#merlin-abs-paths"
```

Comment:
I initially planned to keep it until the appropriate dune release. It was needed for a test that I added that called dune explicitly. 
However I removed this test because it would have caused the same issue as with release 3.3.5.
The pin should not be useful any more, I will remove it.